easybuild/easyconfigs/p/Python/Python-2.7.13-GCCcore-6.4.0-bare.eb

```diff
@@ -0,0 +1,56 @@
+name = 'Python'
+version = '2.7.13'
+versionsuffix = '-bare'
+
+homepage = 'http://python.org/'
+
+description = """
+ Python is a programming language that lets you work more quickly and
+ integrate your systems more effectively.
+
+Note: This module is meant to provide a builddependency for other Python
+      modules while using EasyBuild's --minimaltoolchain option.  Modules
+      built with it will require the full Python later
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '6.4.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+checksums = ['a4f05a0720ce0fd92626f0278b6b433eee9a6173ddf2bced7957dfb599a5ece1']
+
+builddependencies = [
+    ('binutils', '2.28'),
+]
+
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('libreadline', '7.0'),
+    ('ncurses', '6.0'),
+    ('SQLite', '3.19.3'),
+    ('zlib', '1.2.11'),
+]
+
+osdependencies = [
+    # rely upon distribution for timely security updates
+    ('openssl-devel', 'libssl-dev', 'libopenssl-devel'),
+]
+
+# We hide this by default since users should not use it in production,
+# high-performance Python should be delivered at compiler level with
+# a default extension set
+# hidden = True
+
+# bare installation: only known module deps for GCCcore tools included
+exts_list = []
+
+# Until such time that EasyBuild accepts 'rpath' as a toolchainopt...
+# add RPATH so this can be used as a builddependency for "bare" PythonPackages
+preconfigopts = """
+    sed -e\
+      's:$(LINKCC) $(LDFLAGS) $(LINKFORSHARED):$(LINKCC) $(LDFLAGS) $(LINKFORSHARED) -Wl,-rpath=%(installdir)s/lib:'\
+      -i.eb Makefile.pre.in
```

Comment:
@boegel well, AFAIK, there is no way to tell Travis to use --expirmental.  Why should I be FORCED to make three different versions (iomkl/intel/foss) for a module when the GCCcore one will suffice as a builddep that can be used with any of those later?

@damianam because pythonpackage.py doesn't set LD_LIBRARY_PATH that includes builddeps when it does a "python -c 'import 'foo'" check..  So the check fails because python can't find libpythonX.Y.so.  The alternative is disable that check in everything that uses Python-bare, which seems much worse than any concern expressed thus far.

Y'all could fix this by updating pythonpackage.py to set the LD_LIBRARY_PATH for the Python-bare that was used as a builddep.   Not sure if that's the best solution.  I just know this one works).

Like I said, I have some 50+ modules using Python bare.   I"ve spent a lot of time working through the issues.   Multiply that by 2 versions of Python.  Y'all are basically saying I should have to multiply that by 3 (iomkl/intel/foss).   So I end up with 300 modules when 100 would have sufficed.  That's just a waste of time and space.